.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "python.pythonPath": "/home/alex/anaconda3/bin/python",
+    "python.pythonPath": "C:\\Users\\azfar.khoja\\Miniconda3\\envs\\pytorch\\python.exe",
```

Comment:
Both our systems are going to keep changing this.
Could just remove it from git.
If I do it, it would just delete the file at your end.

Reply:
Just leave the file as-is and I'll manage a local branch which has changes applicable to my machine